parent/pom.xml

```diff
@@ -2103,6 +2133,71 @@
                </plugin>
 
                <plugin>
+                  <groupId>org.codehaus.mojo</groupId>
+                  <artifactId>build-helper-maven-plugin</artifactId>
+                  <version>1.7</version>
+                  <executions>
+                     <execution>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                           <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                           <sources>
+                              <source>${dir.ispn}/atomic-factory/src/main/java</source>
+                              <source>${dir.ispn}/cdi/common/src/main/java</source>
+                              <source>${dir.ispn}/cdi/embedded/src/main/java</source>
+                              <source>${dir.ispn}/cdi/remote/src/main/java</source>
+                              <source>${dir.ispn}/checkstyle/src/main/java</source>
+                              <source>${dir.ispn}/cli/cli-client/src/main/java</source>
+                              <source>${dir.ispn}/cli/cli-interpreter/src/main/java</source>
+                              <source>${dir.ispn}/client/hotrod-client/src/main/java</source>
```

Comment:
Thanks for the explanation - I missed it.

I've been thinking about this and I think we should introduce Sonar analysis into the CI loop (it appeared a couple of times on the mailing list). So how about merging it as is for now and if we succeed with Sonar integration - we'll remove whole Jacoco reporting part (Sonar has build-in coverage visualizer). 

@andyuk1986 @tristantarrant @mgencur WDYT?